runtime/lua/vscode/internal.lua

```diff
@@ -280,4 +282,72 @@ function M.wslpath(path)
   return vim.trim(ret)
 end
 
+--#region Buffer management
+
+local function set_buffer_autocmd(buf)
+  api.nvim_create_autocmd({ "BufWriteCmd" }, {
+    buffer = buf,
+    callback = function(ev)
+      local current_name = vim.api.nvim_buf_get_name(ev.buf)
```

Comment:
Out of curiosity, would operating by uri be more robust than by name?

Reply:
The correct one is name, which, like the autocmds data, is data in Vim. It is important to ensure that the same type of data is processed together.